src/scripts/main.js

```diff
@@ -15,14 +15,26 @@ const food = {
       'Banana': {},
       'Pineapple': {},
     },
-
   },
 };
 
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const rootList = document.createElement('ul');
+
+  for (const key in data) {
+    const rootListItem = document.createElement('li');
+
+    rootListItem.innerText = key;
+    rootList.appendChild(rootListItem);
+
+    if (data[key] !== undefined) {
```

Comment:
```suggestion
```
redundant.
instead check before create new list whether data has keys, and if not make early return
it will fix this test and not make empty list
![Screenshot from 2023-05-17 12-34-50](https://github.com/mate-academy/js_tree-from-object-DOM/assets/33029864/19585ce5-c6ea-42fd-a96f-a476dc412810)
![Screenshot from 2023-05-17 12-34-42](https://github.com/mate-academy/js_tree-from-object-DOM/assets/33029864/c57560fe-2be2-4c46-b345-4c862dc700de)

Reply:
didn't fix it